aws-concepts/aws-ses.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Amazon SES (Simple Email Service) | BlueFox Email
+description: AWS SES is a cloud email service provider that helps digital marketers and application developers send marketing, notification, and transactional emails.
```

Comment:
Now, that I read it again, I would just say that AWS SES is a cloud-base email service... I would not use provider, coz AWS is the provider actually.